distributed/nanny.py

```diff
@@ -255,7 +256,7 @@ def __init__(  # type: ignore[no-untyped-def]
         handlers = {
             "instantiate": self.instantiate,
             "kill": self.kill,
-            "restart": self.restart,
+            "restart": self.restart,  # TODO: Is this being used anywhere?
```

Comment:
`Client.restart` uses this handler

Reply:
#6714 removed the call to `Nanny.restart` in `Scheduler.restart` which in turn is used by `Client.restart` (see https://github.com/dask/distributed/pull/6714#issuecomment-1188288371 for the reason behind this decision). I see that #7154 reintroduces using `Nanny.restart`, see https://github.com/dask/distributed/pull/7154#pullrequestreview-1150477772 for my thoughts on that.

https://github.com/dask/distributed/pull/6714/files#diff-bbcf2e505bf2f9dd0dc25de4582115ee4ed4a6e80997affc7b22122912cc6591R5196